erp/provider/search.py

```diff
@@ -70,3 +75,136 @@ def global_search(terms, code_insee):
+            name=translate("Entrée de plain pied ou accessible via rampe ou ascenseur"),
+        ),
+        Equipment(
+            slug="having_entry_min_width",
+            manager=ErpQuerySet.having_entry_min_width,
+            name=translate("Largeur de porte supérieure à 80cm ou donnée inconnue"),
+        ),
+        Equipment(
+            slug="having_adapted_entry",
+            manager=ErpQuerySet.having_adapted_entry,
+            name=translate("Entrée spécifique PMR"),
+        ),
+        Equipment(
+            slug="having_entry_easily_identifiable",
+            manager=ErpQuerySet.having_entry_easily_identifiable,
+            name=translate("Entrée facilement repérable"),
+        ),
+        Equipment(slug="having_sound_beacon", manager=ErpQuerySet.having_sound_beacon, name=translate("Balise sonore")),
+        Equipment(
+            slug="having_visible_reception",
+            manager=ErpQuerySet.having_visible_reception,
+            name=translate("Visibilité de l'accueil depuis l'entrée"),
+        ),
+        Equipment(slug="having_staff", manager=ErpQuerySet.having_staff, name=translate("Présence de personnel")),
+        Equipment(
+            slug="having_trained_staff",
+            manager=ErpQuerySet.having_trained_staff,
+            name=translate("Personnel sensibilisé ou formé"),
+        ),
+        Equipment(
+            slug="having_audiodescription",
+            manager=ErpQuerySet.having_audiodescription,
+            name=translate("Audiodescription"),
+        ),
+        Equipment(
+            slug="having_hearing_equipments",
+            manager=ErpQuerySet.having_hearing_equipments,
+            name=translate("Equipements spécifiques pour personne malentendante"),
+        ),
+        Equipment(
+            slug="having_potentially_all_at_ground_level",
+            manager=ErpQuerySet.having_potentially_all_at_ground_level,
+            name=translate(
+                "Chemin de plain pied jusqu'à l'accueil ou accessible via rampe ou ascenseur ou donnée inconnue"
+            ),
+        ),
+        Equipment(
+            slug="having_entry_no_shrink",
+            manager=ErpQuerySet.having_entry_no_shrink,
+            name=translate("Chemin sans rétrécissement jusqu'à l'accueil ou donnée inconnue"),
+        ),
+        Equipment(
+            slug="having_accessible_rooms",
+            manager=ErpQuerySet.having_accessible_rooms,
+            name=translate("Chambre accessible"),
+        ),
+        Equipment(slug="having_adapted_wc", manager=ErpQuerySet.having_adapted_wc, name=translate("Toilette PMR")),
+        Equipment(slug="having_label", manager=ErpQuerySet.having_label, name=translate("Label")),
+    ]
+
+    if as_dict:
+        return {eq.slug: eq.name for eq in equipments}
+    return equipments
+
+
+def get_equipment_by_slug(slug: str):
+    if not slug:
+        return None
+    equipments = get_equipments()
+    for equipment in equipments:
```

Comment:
not proud of this, and of the global structure of equipments, hard to get_by_slug, etc

Reply:
Why not use the as_dict param and just return equiments.get("slug") ? :thinking: